SecretManagement.KeePass.Extension/Tests/Test-SecretVault.Tests.ps1

```diff
@@ -0,0 +1,60 @@
+get-module *Secret* | Remove-Module -ErrorAction SilentlyContinue
+Import-Module -name Microsoft.PowerShell.SecretManagement
+Microsoft.PowerShell.SecretManagement\Get-SecretVault -Name keepasspestertest* | Microsoft.PowerShell.SecretManagement\Unregister-SecretVault -erroraction SilentlyContinue
+Import-Module -Name "$($PSScriptRoot)\..\secretmanagement.keepass.extension.psd1" -force
+
+InModuleScope -ModuleName 'SecretManagement.KeePass.Extension' {
```

Comment:
InModuleScope only needs to wrap the It statements that run in the module context.